ajax/libs/FiraCode/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "FiraCode",
+  "version": "1.204",
+  "filename": "fira_code.min.css",
+  "description": "Fira Code is an extension of the Fira Mono font containing a set of ligatures for common programming multi-character combinations.",
+  "homepage": "https://github.com/tonsky/FiraCode/wiki",
+  "keywords": [
+    "font",
+    "monospaces",
+    "programming"
```

Comment:
Let's add more keywords `programming-ligatures`, `ligatures`, let it be searched easier.
These keywords show at the blue part.
![image](https://cloud.githubusercontent.com/assets/8349587/26530624/131d3cec-440b-11e7-8d92-8dd1f9f70298.png)